puppet/modules/chassis/files/local-config-hosts.php

```diff
@@ -14,6 +14,17 @@
 function bootstrap() {
 	add_filter( 'wpmu_new_blog', __NAMESPACE__ . '\\add_host' );
 	add_filter( 'delete_blog', __NAMESPACE__ . '\\remove_host' );
+    add_action( 'init', __NAMESPACE__ . '\\maybe_add_host' );
```

Comment:
@BronsonQuick How did I know you were going to link to that video? That is the result of me copying and pasting. Fixing it now.